firebase-firestore/src/main/java/com/google/firebase/firestore/local/RemoteDocumentCache.java

```diff
@@ -80,19 +79,13 @@
   Map<DocumentKey, MutableDocument> getAll(String collectionGroup, IndexOffset offset, int count);
 
   /**
-   * Executes a query against the cached Document entries
+   * Returns the documents from the provided collection.
    *
-   * <p>Implementations may return extra documents if convenient. The results should be re-filtered
-   * by the consumer before presenting them to the user.
-   *
-   * <p>Cached entries for non-existing documents have no bearing on query results.
-   *
-   * @param query The query to match documents against.
+   * @param collection The collection to read.
    * @param offset The read time and document key to start scanning at (exclusive).
    * @return The set of matching documents.
```

Comment:
nit: Update the return value javadocs to specify that the returned Map is a newly-created Map that the caller is free to modify. Previously, when it was `ImmutableSortedMap`, this was clear but now that it returns a mutable `Map` it's not clear without looking at the implementation whether or not it's a copy.

Reply:
Good point. I adjusted the comment.